### DISCUSSION TITLE
recording.importStatus value is "none" for recording.imported webhook

### DISCUSSION THREAD

**Discussion by rg-gravis** [foxglove/community#1041] [Bugs] [answered]
I created a `recording.imported` webhook and triggered it by `cp`-ing an mcap file whose name matched `WATCH_AUTO_IMPORT_PATTERN` into the `STORAGE_ROOT` directory of foxglove-agent. I did see an event dispatched to my webhook endpoint, but it had the content
```json
{
  "attemptedAt": "2024-09-17T10:55:34.820Z",
  "timestamp": "2024-09-17T10:55:34.008Z",
  "webhookEventId": "whev_0dBaHw4Yd7dYpTSc",
  "webhookId": "whk_0dBZSDoeZMoOX1Mb",
  "type": "recording.imported",
  "recording": {
    "id": "wU9jX179u4oQgL1v",
    "path": "correct-filename.mcap",
    "importStatus": "none"
  }
}
```
Which is also what's shown in the "Request Data" tab of the webhook delivery details page. I was expecting the `recording.importStatus` field to have the value `"complete"`, not `"none"`.

The recording itself was successfully auto-imported and appears as expected.

**Comment by bryfox** [comment]
Thanks for the report; it does seem like a bug. Tracking internally as FG-8830.

**Comment by mguida22** [comment, marked as answer]
This is now fixed, thanks again for reporting it.
